src/software/ai/hl/stp/tactic/dribble/dribble_fsm.cpp

```diff
@@ -106,6 +136,33 @@ void DribbleFSM::dribble(const Update &event)
             event.control_params.dribble_destination,
             event.control_params.final_dribble_orientation);
 
+    Point dribble_destination = getDribbleBallDestination(
+        event.common.world.ball().position(), event.control_params.dribble_destination);
+
+    // when we have the ball in the dribbler, only then we pivot around the ball
+    // if (distance(event.common.robot.position(), event.common.world.ball().position()) <
+    // 0.088)
+    if (event.common.robot.isNearDribbler(event.common.world.ball().position()))
```

Comment:
Wouldn't `dribble` action pretty much always run when we have possession? So wouldn't we always go into this if statement? If we do, then how does that affect the `target_destination` to be the orbitting destination or the actual final destination?